### DISCUSSION TITLE
targetdir doesn't seem to work on some configurations

### DISCUSSION THREAD

**Discussion by michael-burrowes-ts** [premake/premake-core#1810] [Q&A]
I have a premake file that prepares settings targeting both Windows and Oculus (android) platforms. Everything seems to work properly, except the android config doesn't seem to get the targetdir set for it. In particular, I'm using the vs2019 action, and using both windows and android configs within vs. I'll see "output directory" set properly when I'm on the windows config, but it's not set when I go to the android one. Wondering if this is a known issue? Maybe an artifact of doing a config with "system 'android'" for vs?

**Comment by mumu-san** [comment]
I found the same issue
premake beta2 \ vs2022 \ system android
